install/kubernetes/cilium/templates/cilium-preflight/poddisruptionbudget.yaml

```diff
@@ -0,0 +1,16 @@
+{{- if and .Values.preflight.enabled .Values.preflight.validateCNPs .Values.preflight.podDisruptionBudget.enabled }}
+apiVersion: policy/v1beta1
+kind: PodDisruptionBudget
+metadata:
+  name: cilium-pre-flight-check
+  namespace: {{ .Release.Namespace }}
+  labels:
+    k8s-app: cilium-pre-flight-check-deployment
+    kubernetes.io/cluster-service: "true"
+spec:
+  maxUnavailable: {{ .Values.preflight.maxUnavailable }}
```

Comment:
should it be `.Values.preflight.podDisruptionBudget.maxUnavailable` instead ?

Reply:
probably a question to the team, I am unable to understand why the default value here is 2 🤔, will be great if anyone can shed some light here.

Reply:
Looking at the history of the old values, I think they were added by mistake (see https://github.com/cilium/cilium/pull/18317#pullrequestreview-837710367). I think the most conservative default of 1 makes sense here.

Reply:
maxUnavailable=1 has been set by default in the latest commit.